addons/website/static/src/components/configurator/configurator.js

```diff
@@ -28,6 +28,19 @@ const WEBSITE_PURPOSES = {
     5: {id: 5, label: _lt("schedule appointments"), name: 'schedule_appointments'}
 };
 
+const PRESET_BG_MATCHING = {
+    1: 4,
+    2: 3,
+    3: 2,
+    4: 1,
+    5: 5,
+};
+
+const DEFAULT_PRESETS = {
+    menu: 1,
+    footer: 5,
+};
```

Comment:
Please, let's try to limit the "usine à gaz" stuff around here :/ Duplicated code, nobody will think about checking this if we ever change the palettes defauts. But most of all I don't think you need this at all from what you are doing in the rest of this PR -> If needed, just print more variable from the CSS to use here in JS (even your whole JS logic to have the final CSS color of the menu/header for a given palette could be done in scss I guess?) :wink:

Reply:
Yes I imagine I can just compute these menu/footer bg color for each palette in scss, print them as variable in the CSS and retrieve with getCSSVariableValue. Indeed much better.